plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/DBLookupMediatorPropertiesEditionPartForm.java

```diff
@@ -211,12 +226,13 @@ public Composite createFigure(final Composite parent, final FormToolkit widgetFa
 	 * 
 	 */
 	public void createControls(final FormToolkit widgetFactory, Composite view) {
-	 // Start of user code for loading
+	 
 	    CompositionSequence dBLookupMediatorStep = new BindingCompositionSequence(propertiesEditionComponent);
 	    
 	    CompositionStep connectionStep = dBLookupMediatorStep.addStep(EsbViewsRepository.DBLookupMediator.Connection.class);
-        connectionStep.addStep(EsbViewsRepository.DBLookupMediator.Connection.connectionType);
-        connectionStep.addStep(EsbViewsRepository.DBLookupMediator.Connection.connectionDsType);
+	    connectionStep.addStep(EsbViewsRepository.DBLookupMediator.Connection.connectionType);
```

Comment:
Shall we fix the code formatting issues here?

Reply:
fixed